localstack-core/localstack/services/events/rule.py

```diff
@@ -38,7 +52,7 @@ def __init__(
         event_pattern: Optional[EventPattern] = None,
         state: Optional[RuleState] = None,
         description: Optional[RuleDescription] = None,
-        role_arn: Optional[RoleArn] = None,
+        role_arn: Optional[Arn] = None,
```

Comment:
`RoleArn` or `Arn`?

Reply:
RoleAnr = Arn both are type String and String = str - not sure why this complicated

Reply:
It doesn't matter that they are both of type str. Typing intends to distinguish input/output/fields by the exact type. In this example, proper typing could highlight if we try to assign any random ARN to a role ARN. It's there to help that only the variable of the correct type can be assigned.

Reply:
mypy might be worth checking out to demonstrate what a type checker can offer: https://github.com/python/mypy

Reply:
fixed